includes/class-wcpdf-admin.php

```diff
@@ -1391,14 +1262,87 @@ public function sort_orders_by_numeric_invoice_number( $query ): void {
      *
 	 * @return bool
 	 */
-	private function is_invoice_number_numeric() {
+	private function is_invoice_number_numeric(): bool {
 		$invoice_settings = WPO_WCPDF()->settings->get_document_settings( 'invoice' );
 		$is_numeric       = ( empty( $invoice_settings['number_format']['prefix'] ) || ctype_digit( $invoice_settings['number_format']['prefix'] ) ) &&
 		                    ( empty( $invoice_settings['number_format']['suffix'] ) || ctype_digit( $invoice_settings['number_format']['suffix'] ) );
 
 		return apply_filters( 'wpo_wcpdf_invoice_number_is_numeric', $is_numeric );
 	}
 
+	/**
+	 * Append ready documents to the heartbeat response.
+	 *
+	 * @param array $response
+	 * @param array $data
+	 *
+	 * @return array
+	 */
+	public function handle_heartbeat_received( array $response, array $data ): array {
+		if ( empty( $data['wpo_pending_documents'] ) || empty( $data['wc-refresh-order-lock'] ) ) {
```

Comment:
I used the `||` in the condition to prevent the execution of the rest of the function if either is empty. 
So this should early return if the `$data['wc-refresh-order-lock']` is empty and the `$data['wpo_pending_documents']` is not. Wouldn't it?

Reply:
You're right, my bad.